src/prof/src/lib.rs

```diff
@@ -7,15 +7,22 @@
 // the Business Source License, use of this software will be governed
 // by the Apache License, Version 2.0.
 
-use std::ffi::CString;
 use std::os::unix::ffi::OsStrExt;
-use std::sync::{Arc, Mutex};
-use std::time::Instant;
+use std::sync::Arc;
```

Comment:
Since there are a bunch of `pub` items in this module could you add a doc-comment mentioning what the primary entrypoint is?

Reply:
Yeah, will do